src/artifacts.js

```diff
@@ -0,0 +1,62 @@
+import { join, dirname } from 'path'
+import { readdir, stat, unlink } from 'fs'
+import { readFileAsync, writeFileAsync } from './util'
+import Bluebird from 'bluebird'
+import mkdirp from 'mkdirp'
+
+const { promisify, map } = Bluebird
+const mkdirpAsync = promisify(mkdirp)
+const statAsync = promisify(stat)
```

Comment:
This is in utils.  Check around for more dupes like this one.  It's super easy to duplicate things like this.